generalize workchain sphinx extension to processes (#3314)
aiidateam/aiida-core: aiida/sphinxext/tests/workchain_source/index.rst

    :module: demo_workchain

> If you don't want to show only the inputs that are stored in the database, use the ``:hide-unstored-inputs:`` option.

Reviewer: ```suggestion
If you want to hide the inputs that are not stored as nodes in the database, use the ``:hide-unstored-inputs:`` option.
```
